README.md

```diff
@@ -1 +1,51 @@
-# library
\ No newline at end of file
+## Library App
+
```

Comment:
- [ ] Let's add a screenshot of your book library to the README.md file.

Reply:
Screenshot has been added.